src/wms/src/test/resources/org/geoserver/wms/content_json.ftl

```diff
@@ -0,0 +1,26 @@
+<#list features as feature>
+{
+"content" : "this is the content",
+"type": "Feature",
+"id" : "${feature.fid}",
+"geometry_name":"the_geom",
+<#list feature.attributes as attribute1>
+<#if attribute1.isGeometry>
+"geometry": "${attribute1.value}",
```

Comment:
Is this going to encode a valid GeoJSON geometry? How does it work?

Reply:
Tried interactively, it does not seem to generate valid GeoJSON:

```
{
header: "this is the header",
type: "FeatureCollection",
features: [
{
content: "this is the content",
type: "Feature",
id: "archsites.7",
geometry_name: "the_geom",
geometry: "POINT (606635 4920773)",
properties: {
cat: "7",
str1: "Jensen Pass"
}
}
],
footer: "this is the footer"
}
```
My guess is that a function to encode a geometry as a valid GeoJSON geometry is missing.

Reply:
fixed